src/main/java/CarRacing/CarRacing.java

```diff
@@ -0,0 +1,55 @@
+package CarRacing;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class CarRacing {
+
+    public static final int BOUND = 10;
+    public static final int MIN_MOVE_VALUE = 4;
+
+    private static final Random random = new Random();
+
+    public boolean shouldMove(int randomValue) {
+        return randomValue >= MIN_MOVE_VALUE;
+    }
+
+    public Map<Integer, Integer> race(int carNums, int moveNums) {
```

Comment:
객체지향 생활 체조 원칙 중 `규칙 5: 줄여쓰지 않는다(축약 금지).` 항목을 참고해주세요.
```suggestion
    public Map<Integer, Integer> race(int carNumbers, int moveNumbers) {
```

Reply:
수정했습니다. 습관이 무섭네요 ㅠㅠ